docs/getting_started/docker.md

```diff
@@ -0,0 +1,78 @@
+# Running Labelmerge with Docker on Windows
+
+**Note, these instructions assume you have Docker already installed on a Windows system.
```

Comment:
Hmm, not sure what formatting this was meant to be - maybe italicized? I don't think the way `**` is used here will actually result in any sort of formatting and will be rendered as is.